gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobInfo.java

```diff
@@ -158,11 +173,20 @@ Builder componentCount(Integer componentCount) {
       return this;
     }
 
+    /**
+     * Sets the blob's data cache control.
```

Comment:
Would be nice to include the reference to the valid structure.
e.g. in this case refer to https://tools.ietf.org/html/rfc7234#section-5.2.
See [this](https://cloud.google.com/storage/docs/json_api/v1/objects/insert) for example that includes links for other settings such as content-Type, Content-Disposition,...